app/models/manageiq/providers/vmware/infra_manager/vm_or_template_shared/scanning.rb

```diff
@@ -29,10 +43,6 @@ def perform_metadata_sync(ost)
     sync_stashed_metadata(ost)
   end
 
-  def validate_smartstate_analysis
-    validate_supported_check("Smartstate Analysis")
```

Comment:
Added back the common method with modifications as required to remove the redundant code.